src/evict/evict_lru.c

```diff
@@ -1720,10 +1720,22 @@ __evict_walk_file(WT_SESSION_IMPL *session,
 			 * Try a different walk start point next time if a
 			 * walk gave up.
 			 */
-			btree->evict_walk_state =
```

Comment:
@michaelcahill Do you mind being the arbiter? I don't feel super strongly - it's possible that we'll end up choosing walk start point strategies in a more sophisticated fashion in the future which would make this moot.